HtmlRenderer/Renderer.cs

```diff
@@ -28,6 +28,7 @@ public class Renderer : IDisposable
 
         private BrowserWrapper _browser;
         private bool _isWindowless;
+        private bool isStopRender = true;
```

Comment:
Should be `_isStopRender` to match other private variables, and IMO `_isRendererStopped` is a better name